docs/sources/configuration/_index.md

```diff
@@ -851,28 +851,31 @@ The `swift_storage_config` configures Swift as a general storage for different d
 [container_name: <string> | default = "cortex"]
 ```
 
-## hedging_config
+## hedging
```

Comment:
We should standardise on whether we include the `_config` suffix; it's a bit inconsistent right now. For example, the few lines above this:

```yaml
  # Configures backend rule storage for Azure.
  [azure: <azure_storage_config>]
  # Configures backend rule storage for GCS.
  [gcs: <gcs_storage_config>]
  # Configures backend rule storage for S3.
  [s3: <s3_storage_config>]
  # Configures backend rule storage for Swift.
  [swift: <swift_storage_config>]
```

I agree with your approach, but perhaps we can correct this in a subsequent PR?

Reply:
Definitely, @dannykopping . I was attempting to only address hedging-related configuration in this PR.